lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1494,6 +1494,15 @@ static PrimDictConstructOp getDictConstructIfNotModified(Value torchDict) {
   return torchDict.getDefiningOp<Torch::PrimDictConstructOp>();
 }
 
+static PrimListConstructOp getListConstructIfNotModified(Value torchList) {
```

Comment:
I think it would make things more clear to call this `static bool isListConstructNotModified(Value torchList)`, since the return value is not used by the fold below